### PR TITLE
Move more operators to claim 4.12 clusters

### PR DESCRIPTION
**Pull request by viroel**

### CODE REVIEW

**Comment by viroel** [comment]
/hold

under testing

**Comment by viroel** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-manila-operator-main-manila-operator-build-deploy-tempest pull-ci-openstack-k8s-operators-cinder-operator-main-cinder-operator-build-deploy-kuttl pull-ci-openstack-k8s-operators-cinder-operator-main-cinder-operator-build-deploy-tempest

**Comment by Sandeepyadav93** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-cinder-operator-main-cinder-operator-build-deploy-kuttl

File: ci-operator/config/openstack-k8s-operators/cinder-operator/openstack-k8s-operators-cinder-operator-main.yaml [hunk 1]
```diff
@@ -77,7 +79,7 @@ tests:
     owner: openstack-k8s-operators
     product: ocp
     timeout: 1h0m0s
-    version: "4.10"
+    version: "4.12"
   steps:
     workflow: openstack-k8s-operators-test-build-deploy-kuttl
```
Comment (Sandeepyadav93) [inline ci-operator/config/openstack-k8s-operators/cinder-operator/openstack-k8s-operators-cinder-operator-main.yaml:84]: @viroel 

we missed to add `CEPH_HOSTNETWORK: "false"` here.

~~~
    env:
      CEPH_HOSTNETWORK: "false"
~~~

that's why we hit below issue in the first run

https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/41611/rehearse-41611-pull-ci-openstack-k8s-operators-cinder-operator-main-cinder-operator-build-deploy-kuttl/1683637428847382528/artifacts/cinder-operator-build-deploy-kuttl/openstack-k8s-operators-gather/artifacts/api-resources/openstackcontrolplanes.yaml

~~~
    - lastTransitionTime: "2023-07-25T01:48:56Z"
      message: OpenStackControlPlane Glance in progress
      reason: Requested
      severity: Info
      status: "False"
      type: Ready
~~~

**Review by Sandeepyadav93 — CHANGES_REQUESTED** [review]

File: ci-operator/config/openstack-k8s-operators/cinder-operator/openstack-k8s-operators-cinder-operator-main.yaml (hunk 1 above)
Comment (Sandeepyadav93) [reply ci-operator/config/openstack-k8s-operators/cinder-operator/openstack-k8s-operators-cinder-operator-main.yaml:86]: fixed.

**Comment by Sandeepyadav93** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-glance-operator-main-glance-operator-build-deploy-kuttl

**Comment by Sandeepyadav93** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-glance-operator-main-glance-operator-build-deploy-tempest

**Pull request merged**